src/bp-forums/common/template.php

```diff
@@ -1617,7 +1617,7 @@ function bbp_get_dropdown( $args = '' ) {
 	// Items found so walk the tree
 	if ( ! empty( $posts ) ) {
 		add_filter( 'list_pages', 'bbp_reply_attributes_meta_box_discussion_reply_title', 999, 2 );
-		unset( $r['walker'] );
+		// unset( $r['walker'] );
```

Comment:
@asaquzzamanbb  make it conditionally where you needed.

Reply:
@KartikSuthar 
Make it conditional here https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/73e8af6bd32605bb124c2c27700db51e1c1517a7